extensions/wikia/CommunityPage/scripts/entryPoint.js

```diff
@@ -10,14 +10,15 @@ require([
 		trackingMethod: 'analytics'
 	});
 
-	function initTracking() {
-		// Track clicks in contribution module
-		$('.community-page-entry-point-module').on('mousedown touchstart', 'a', function () {
-			track({
-				label: 'enter-button-click',
-			});
+	$('.community-page-entry-point-module').on('mousedown touchstart', 'a', function () {
```

Comment:
I think we can bind event to '.community-page-entry-point-module .entry-point-button' directly